sharkfuser/tests/lit/test_conv_grouped_asm_emitter_nhwc_kcrs.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright 2025 Advanced Micro Devices, Inc.
```

Comment:
There's no such thing as "too many tests" but let's make sure they add value. If they're redundant, let's drop it.